@AH-64D Apache Longbow/Addons/fza_ah64_weapons/functions/pylon/fn_SetPylonOwner.sqf

```diff
@@ -0,0 +1,24 @@
+/* ----------------------------------------------------------------------------
+Function: fza_weapons_fnc_SetPylonOwner
+
+Description:
+	
+Parameters:
+	
+Returns:
+	...
+
+Examples:
+	...
+
+Author:
+	Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params ["_heli","_Wassing"];
+
+_wassing = "rockets"; 
+_turret = -1; 
+_PylonInfo = getAllPylonsInfo _heli; 
+if (_heli turretLocal [-1]) then {_turret = 0;}; 
+[_heli, _wassing,_PylonInfo] remoteExec ["fza_weapons_fnc_SetPylonOwnerInternal", _heli turretUnit [_turret]]; 
+[_heli, _wassing,_PylonInfo] call fza_weapons_fnc_SetPylonOwnerInternal;
```

Comment:
Does this assume that the player is in a certain seat? If so, which? Does it work when the player is in both?